testdata/stats/client_config_customized.yaml.tmpl

```diff
@@ -1,8 +1,5 @@
 field_separator: ";.;"
 definitions:
-- name: requests_total
```

Comment:
Why is requests_total deleted?

Reply:
This test case was changing the value for a standard metric. I don't think we want to support this feature.